webapp/channels/src/components/team_groups_manage_modal/team_groups_manage_modal.test.tsx

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {shallow} from 'enzyme';
```

Comment:
there is an ongoing effort to use react-testing-library instead of enzyme. To facilitate that, we have created a set of utils functions you can find here: https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/tests/react_testing_utils.tsx more details here: https://developers.mattermost.com/contribute/more-info/webapp/unit-testing/

Reply:
Sure, I am now using the same in my updated commits on this PR.